agent/newrelic-install.sh

```diff
@@ -120,7 +120,9 @@ error() {
 # Get the OS type.
 #
 ostype=
-if [ -f /etc/redhat-release -o -f /etc/redhat_version ]; then
+if [ -f /proc/version ] && [ -n "$(grep 'Alpine' /proc/version)" ]; then
+  ostype=alpine
```

Comment:
I'm not sure this will work. Here's what I got on centos6:
```
docker run --platform linux/amd64 centos:6 grep 'Alpine' /proc/version
Linux version 5.10.76-linuxkit (root@buildkitsandbox) (gcc (Alpine 10.2.1_pre1) 10.2.1 20201203, GNU ld (GNU Binutils) 2.35.2) #1 SMP PREEMPT Mon Nov 8 11:22:26 UTC 2021
```
I think the check in line 129 is the correct method to detect if script is running on alpine distribution:
```
docker run --platform linux/amd64 alpine:latest cat /etc/alpine-release
3.15.1
```
The problem of incorrect detection of alpine stems from the fact that check for alpine is executed after the check if `/etc/dpkg` directory exists, which IMHO is not the correct way to determine which Linux distribution the script is running on. I think to correctly address this issue the check in line 127 has to be modified.

Reply:
@mfulb lsb_requires program requires additional package to be installed. It seems like /etc/os-release becomes a new standard. [Here's](http://0pointer.de/blog/projects/os-release.html) some more background why using /etc/os-release to determine Linux distribution may be a better option than depending on external utility like lsb_release. A little more info on what is in os-release file: [man os-release](https://man7.org/linux/man-pages/man5/os-release.5.html).

Reply:
Done [fb2273ea](https://github.com/newrelic/newrelic-php-agent/pull/391/commits/fb2273ea82f05eee4b94005adb9a41c1c8b5b2e1).